securepy/stdcapture.py

```diff
@@ -121,23 +125,24 @@ def __call__(self, func: t.Callable) -> t.Any:
         Return value will be the original return from `func`.
 
         STDOUT & STDERR will be captured and can be obtained by doing
-        `StdCapture.stdout`/`StdCapture.stderr`.
+        `IOCage.stdout`/`IOCage.stderr`.
 
-        The functionality is handeled in `StdCapture.capture`, this method
+        The functionality is handeled in `IOCage.capture`, this method
         serves only as a decorator for given `func`.
         """
         @wraps(func)
         def inner(*args, **kwargs) -> t.Any:
             return self.capture(func, args, kwargs)
         return inner
 
-    def capture(self, func: t.Callable, args=None, kwargs=None) -> t.Any:
+    def capture(self, func: t.Callable, args=None, kwargs=None, stdin=None) -> t.Any:
```

Comment:
There should be proper type hinting for `stdin` and it should come before `args` and `kwargs` since those tend to come last in most of the functions.
```suggestion
    def capture(self, func: t.Callable, stdin: t.Optional[str] = None, args=None, kwargs=None) -> t.Any:
```